src/ControlSystem/ControlErrors/Size/ComovingCharSpeedDerivative.hpp

```diff
@@ -0,0 +1,347 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+class DataVector;
+namespace Frame {
+struct Distorted;
+struct Inertial;
+}  // namespace Frame
```

Comment:
Move `Frame` declarations to cpp.

Reply:
I think I need them now in hpp since I use explicit frames.